blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -40,9 +40,11 @@ export default class Pipelines extends Component {
                             { organization && ' / ' }
                             { organization && orgLink }
                         </h1>
-                        <a target="_blank" className="btn-secondary inverse" href={newJobUrl}>
-                            New Pipeline
-                        </a>
+                        <Extensions.Renderer extensionPoint="jenkins.pipeline.create.action">
```

Comment:
Yup. If we need it to be additional rather than replacement we can at some point, just means a bit more complexity keep track of what's a plugin and what's not.